dbt/models/iasworld/schema/iasworld.dweldat.yml

```diff
@@ -920,7 +968,11 @@ sources:
               - accepted_values:
                   name: iasworld_dweldat_char_renovation_in_accepted_values
                   values: ['0', '1']
+                  additional_select_columns: *select-columns
                   config: *unique-conditions
+                  meta:
+                    description: >
+                      user3 (property renovation indicator) should be '0' or '1'
```

Comment:
**suggestion:** It might be useful to update the user field tests to reflect how they're displayed on the iasWorld dwellings tab. For example, this one would be:

```suggestion
                      user3 (Renovated) should be '0' or '1'
```

Reply:
Good idea, I opened up https://github.com/ccao-data/data-architecture/issues/354 to track this work as a separate issue since it'll be a bit of a slog.